src/wp-content/themes/twentytwentythree/style.css

```
@@ -0,0 +1,16 @@
+/*
+Theme Name: Twenty Twenty-Three
+Theme URI: https://github.com/WordPress/twentytwentythree
+Author: wordpressdotorg
+Author URI: https://wordpress.org
+Description:
+Requires at least: 5.9
+Tested up to: 6.1
+Requires PHP: 5.6
+Version: 0.0.1
```

For the initial version, let's use 1.0 as that's what has been used historically when themes are initially merged. Default themes also only use two digits, so this can be just X.Y.

```suggestion
Version: 1.0
```